src/options.rs

```diff
@@ -30,13 +30,20 @@ static CONNECTION_ARG_HINT: &str = "\
     Run `edgedb project init` or use any of `-H`, `-P`, `-I` arguments \
     to specify connection parameters. See `--help` for details";
 
+const CONN_OPTIONS_GROUP: &str =
+    "CONNECTION OPTIONS (`edgedb --help-connect` to see the full list)";
+
+const EDGEDB_ABOUT: &str = "\
+    The `edgedb` command allows to open a REPL to the database,\n\
+    manage projects, spin up local instances, create migrations,\n\
+    and many other things.\n\
+    \n\
+    Running `edgedb` without a subcommand opens a REPL.";
```

Comment:
How widespread is the understanding of REPL?  Python and Postgres call theirs "interactive shell" or "interactive terminal".

Reply:
You have a point. "Interactive shell" sounds way better than "interactive front-end" though.